README.md

```diff
@@ -51,19 +51,20 @@ To install `uv`, follow the [UV Installation Guide](https://docs.astral.sh/uv/ge
 
 
 ```shell
-uv venv openr1 --python 3.11 && source openr1/bin/activate && uv pip install --upgrade pip --link-mode=copy
```

Comment:
This `--link-mode` flag is a bit specific to the HF cluster and can be resolved by adding `export UV_LINK_MODE=copy` to ones `.bashrc` file.